contracts/oss.json

```diff
@@ -39,6 +39,10 @@
       "name": "Templates",
       "description": "Export and apply InfluxDB **templates**.\nManage **stacks** of templated InfluxDB resources.\n\nInfluxDB templates are prepackaged configurations for\neverything from dashboards and Telegraf to notifications and alerts.\nUse InfluxDB templates to quickly configure a fresh instance of InfluxDB,\nback up your dashboard configuration, or share your configuration with the\nInfluxData community.\n\nUse the `/api/v2/templates` endpoints to export templates and apply templates.\n\n**InfluxDB stacks** are stateful InfluxDB templates that let you\nadd, update, and remove installed template resources over time, avoid duplicating\nresources when applying the same or similar templates more than once, and\napply changes to distributed instances of InfluxDB OSS or InfluxDB Cloud.\n\nUse the `/api/v2/stacks` endpoints to manage installed template resources.\n\n#### Related guides\n\n- [InfluxDB stacks]({{% INFLUXDB_DOCS_URL %}}/influxdb-templates/stacks/)\n- [InfluxDB templates]({{% INFLUXDB_DOCS_URL %}}/influxdb-templates/)\n"
     },
+    {
+      "name": "Users",
+      "description": "Manage users for your organization.\nUsers are those with access to InfluxDB.\nTo grant a user permission to access data, add them as a member of an\norganization and provide them with an API token.\n\n#### User sessions with authorizations\n\nOptionally, you can scope an authorization (and its API token) to a user.\nIf a user signs in with username and password, creating a _user session_,\nthe session carries the permissions granted by all the user's authorizations.\nTo create a user session, use the [`POST /api/v2/signin`](#operation/PostSignin) endpoint.\n\n#### Related guides\n\n- [Manage users]({{% INFLUXDB_DOCS_URL %}}/influxdb/latest/users/).\n- [Create a token scoped to a user]({{% INFLUXDB_DOCS_URL %}}/latest/security/tokens/create-token/#create-a-token-scoped-to-a-user).\n"
```

Comment:
with access to your organization in InfluxDB?

Reply:
add them as a member of an organization >> add them to an organization

Reply:
provide them with an API token that has permissions to read or write to specified resources in your organization

Reply:
Think it may be helpful to be explicit about the relationship between authorization and API token here. 
If I specify permissions for an API token, and a user that I gave the token to logs into session with username/password, are they logging in with the token I gave them?
Does "session carries the permissions granted by all the user's authorizations" mean the same thing as "session carries all permissions granted by the token a user logs in with"?

Reply:
> Think it may be helpful to be explicit about the relationship between authorization and API token here. If I specify permissions for an API token, and a user that I gave the token to logs into session with username/password, are they logging in with the token I gave them? Does "session carries the permissions granted by all the user's authorizations" mean the same thing as "session carries all permissions granted by the token a user logs in with"?

No. To create a session, a user logs in with username and password. In the session, they have all the permissions from all the tokens that are scoped to their userid.